db/update_project_schema.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE project_schema
+    SET project_id = $2, table_name_id = $3, column_name = $4, schema_type_id = $5, size_data = $6, is_primary_key = $7, is_foreign_key = $8, is_serial = $9, is_not_null = $10, is_unique = $11
```

Comment:
I don't believe we need to reset the project_id, that is static once the row is initially created